tests/service.toml

```diff
@@ -51,6 +51,7 @@ type = "bool"
 
 [pairs.storage_class]
 type = "string"
+defaultable = true
```

Comment:
It looks like `WithDefaultStorageClass` is not generated.

Reply:
It has been generated in https://github.com/beyondstorage/go-storage/pull/730/files#diff-d872ad7215a358bcc5ae66b3577ab2db501f7f8497ac6624e7374a19b7f69fedR88. We pointed it as a defaultable pair in namespace before, so we can't find it in this change.